content/docs/concepts/resources/functions.md

```diff
@@ -203,18 +356,18 @@ def get_kubeconfig(self,
 <div>
 <pulumi-choosable type="language" values="java">
 
-(no example available for Java)
+No example available for Java
 
 </pulumi-choosable>
 </div>
 <div>
 <pulumi-choosable type="language" values="yaml">
 
-(no example available for YAML)
+No example available for YAML
 
 </pulumi-choosable>
 </div>
 
 </pulumi-examples></div>
 
-Unlike provider functions, methods always take `Input` arguments, and return an `Output`. Methods do not have invoke options.
+Unlike provider functions, methods always appear in the _output form_: they take `Input` arguments, and return an `Output`. Moreover, methods do not accept invoke options.
```

Comment:
>Moreover, methods do not accept invoke options.

Just curious -- what does it actually mean that "methods don't accept invoke options"? If "invoke options" are themselves just function arguments, and methods do take arguments, what is this sentence supposed to convey to me as a user? If we deleted it, would anything useful be lost?

Reply:
So for a "normal" invoke you can do something like `aws.getAmi(..., { provider: myCustomProvider })`, for instance, where `{ provider: myCustomProvider }` are the invoke options (akin to resource options). Methods don't allow this, (presumably -- again not an expert here) because they'll inherit the options from the resource on which they are being called (so e.g. we don't want to allow you to overload the provider since the owning resource will have already defined one). So at a language level, yes, the method takes arguments, but only those which are inputs to the function, not the "second set" that let you control bits around how Pulumi performs the invoke. If that makes sense? Happy to make changes as you see fit on this.

Reply:
Yes, it does make sense, thanks! I didn't realize it was this "second set" that this referred to.